Snabble/Core/API/SnabbleAPI.swift

```diff
@@ -358,7 +381,7 @@ extension SnabbleAPI {
 
 extension SnabbleAPI {
     static var serverName: String {
-        switch config.baseUrl {
+        switch config.environment.urlString {
```

Comment:
Um die Stringkonstanten hier nicht zu wiederholen sollte das hier in `SnabbleAPI.Environment` umziehen und da die cases benutzen.

Reply:
5b4a74c4bfdbb875af65d8944e3ce4a1969720a5

Habe es leicht anders gemacht, guck mal ob es so auch passt.

Reply:
Den Namen `prod` dürfen wir nicht ändern.